pkg/repo/index.go

```diff
@@ -107,10 +107,9 @@ func (i IndexFile) Add(md *chart.Metadata, filename, baseURL, digest string) {
 	u := filename
 	if baseURL != "" {
 		var err error
-		_, file := filepath.Split(filename)
```

Comment:
looks like removing this line caused the tests to fail in TestIndexAdd @AmandaCameron